README.md

```diff
@@ -47,20 +47,24 @@ cordova platform add --link /path/to/cordova-ios
 
 ## Updating a Cordova project
 
-When you install a new `cordova-cli` version that comes with a new iOS platform version, from within your project:
+When you install a new version of the [`Cordova CLI`](https://www.npmjs.com/package/cordova) that pins a new version of the [`Cordova-iOS`](https://www.npmjs.com/package/cordova-ios) platform, you can follow these simple upgrade steps within your project:
```

Comment:
I think it's better to reference `Cordova-iOS` as the package name literally, `cordova-ios`, which also makes this consistent with other parts of the docs, such as:

> link the appropriate cordova-ios platform folder path:

```suggestion
When you install a new version of the [`Cordova CLI`](https://www.npmjs.com/package/cordova) that pins a new version of the [`cordova-ios`](https://www.npmjs.com/package/cordova-ios) platform, you can follow these simple upgrade steps within your project:
```

Reply:
This wording is to match with the code review changes done by @erisu in PR apache/cordova-android#1308 (see https://github.com/apache/cordova-android/pull/1308/commits/993737e0986f20893f37ebbdb864db0682ecbfc6).

Do you confirm I should reword it to `cordova-ios` here? and do it too for cordova-android through a new PR?

Reply:
Ah well, I'm not really sure :sweat_smile:  and I can't really find an any document to point to that defines how things should be worded/referenced.

So we'll just say I'm wrong and we can follow the style as requested from Erisu.